CHANGES.rst

```diff
@@ -187,6 +187,9 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Override ``HDUList.copy()`` to return a copy instead
+  of ``list``. [#7218]
```

Comment:
I would suggest something like *"... to return a HDUList instance, and do a deep copy by default"*.

Reply:
OK, I've addressed your review comments.